webapi/src/main/scala/org/knora/webapi/messages/v2/responder/listsmessages/ListsMessagesV2.scala

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright © 2015-2018 the contributors (see Contributors.md).
+ *
+ * This file is part of Knora.
+ *
+ * Knora is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Knora is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.knora.webapi.messages.v2.responder.listsmessages
+
+import org.knora.webapi._
+import org.knora.webapi.messages.admin.responder.listsmessages.{ListADM, ListNodeADM, ListNodeInfoADM}
+import org.knora.webapi.messages.admin.responder.usersmessages.UserADM
+import org.knora.webapi.messages.store.triplestoremessages.StringLiteralSequenceV2
+import org.knora.webapi.messages.v2.responder.{KnoraRequestV2, KnoraResponseV2}
+import org.knora.webapi.util.IriConversions._
+import org.knora.webapi.util.StringFormatter
+import org.knora.webapi.util.jsonld._
+
+
+/**
+  * An abstract trait representing a Knora v2 API request message that can be sent to `StandoffResponderV2`.
+  */
+sealed trait ListsResponderRequestV2 extends KnoraRequestV2
+
+/**
+  * Requests a list. A successful response will be a [[ListsGetResponseV2]]
+  *
+  * @param listIri        the IRI of the list (Iri of the list's root node).
+  * @param requestingUser the user making the request.
+  */
+case class ListsGetRequestV2(listIri: IRI,
+                             requestingUser: UserADM) extends ListsResponderRequestV2
+
+
+/**
+  * An abstract trait providing a convenience method for language handling.
+  */
+trait ListResponderResponseV2 {
+
+    /**
+      * Given an Iri and a [[StringLiteralSequenceV2]], gets he string value in the user's preferred language.
+      *
+      * @param iri        the Iri pointing to the string value.
```

Comment:
Missing parameter descriptions

Reply:
done in https://github.com/dhlab-basel/Knora/pull/852/commits/8bba915dd4a23512b68d5dde430551ac7de5c82c